sdk/python/feast/feature_view.py

```diff
@@ -50,28 +50,36 @@
 
 class FeatureView(BaseFeatureView):
     """
-    A FeatureView defines a logical grouping of serveable features.
+    A FeatureView defines a logical group of features.
 
-    Args:
-        name: Name of the group of features.
-        entities: The entities to which this group of features is associated.
+    Attributes:
+        name: The unique name of the feature view.
+        entities: The list of entities with which this group of features is associated.
         ttl: The amount of time this group of features lives. A ttl of 0 indicates that
             this group of features lives forever. Note that large ttl's or a ttl of 0
             can result in extremely computationally intensive queries.
         batch_source: The batch source of data where this group of features is stored.
         stream_source (optional): The stream source of data where this group of features
             is stored.
-        features (optional): The set of features defined as part of this FeatureView.
```

Comment:
Shouldn't we continue to mark optional params as such?

Reply:
good catch - I removed this since `features` is an attribute, and it's always defined

if you want, I can add a docstring to the `__init__` functions, which would then mark `features` as an optional parameter

Reply:
I think that'd be good yeah

Reply:
@achals added docstrings